### ISSUE TITLE
[FEATURE] Logic for thumbnails generation size

### ISSUE THREAD

**Issue by hucurve** [stumpapp/stump#533]
The cause was seeing thumbnails of images that varied greatly in clarity
![image](https://github.com/user-attachments/assets/5fbd63ab-ced7-4e5c-9bbb-09f5e81b658e)

I'm thinking that since the height of the picture is the height of the frame in the UI,
would it be better to limit the  height in a value and scale the image weight equally?
There is a very ugly diagram I do not know if I understand it right.
![image](https://github.com/user-attachments/assets/c008e5bd-4188-4b8f-b409-008295893c83)

**Comment by aaronleopold** [comment]
Just to make sure I'm understanding properly, is the ask to make a third option for thumbnail generation which allows you to set an explicit size for one dimension and just scale the other one?

**Comment by hucurve** [comment]
Yes, of course you can also have new designs for your own design interactions

**Comment by aaronleopold** [comment]
Okay, yeah I think that option makes sense. Thanks for the suggestion! I'll be sure to update this issue once I can get to it

**Comment by aaronleopold** [comment]
@hucurve I have completed this in my branch with all the big migration changes. Here is a snippet of what it looks like:

<img width="1803" height="1278" alt="Image" src="https://github.com/user-attachments/assets/28ffc228-2425-4916-8976-50460297088d" />

Not 100% happy with the UI, but the form elements are what really matters here. I'll keep this open until the migration work is complete 👍

**Comment by aaronleopold** [comment]
I'm closing this issue to clear the backlog a bit, but please note it won't be released until the SeaORM migration is stable
